### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by EyasuTeshome**
## Initialize Project

- Initialize React app.
- Add React Redux
- Structure the application files using a "feature folder" approach and use the ducks pattern for Redux files.
- Add React Router and set two `<Route>`s and `<link>`s for the app's navigation.
- Set Books to the default view.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
-# BookStore
\ No newline at end of file
+## Math Magicians
```
Comment (uwadonat) [inline README.md]: - [ ] I think the Title here should be `BookSore`

File: src/components/Categories.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <h2>Under construction</h2>
+  </div>
```
Comment (uwadonat) [inline src/components/Categories.js]: - [ ] According to the project specifications this components should display only a button with text "Check status". Kindly fix this.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @EyasuTeshome,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Well organized and clean project ✔️
- No linter errors ✔️
- Descriptive pull request message ✔️
almoo


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @EyasuTeshome  👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️ 

Your linter checks are currently not passing. Please fix this 
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by EyasuTeshome** [comment]
@Lucas-Masaba I have fixed the linter error

**Review by cwaku — APPROVED** [review]
Hi @EyasuTeshome 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**